### PR TITLE
Fix printing of txFeePolicy in genesis data

### PR DESCRIPTION
**Pull request by edolstra**
As reported by @sjmackenzie (https://github.com/input-output-hk/rust-cardano/pull/449#commitcomment-32398842).

### CODE REVIEW

**Comment by NicolasDP** [comment]
@edolstra , can you rebase ? the latest merged PR fixed issue with the tower-* dependency that prevented our CI to work

**Comment by edolstra** [comment]
Done.

File: exe-common/examples/generate_genesis_data.rs [hunk 1]
```diff
@@ -97,10 +97,7 @@ fn main() {
         start_time: SystemTime::UNIX_EPOCH + Duration::from_secs(1548089245),
         slot_duration: Duration::from_millis(20000),
         protocol_magic,
-        fee_policy: fee::LinearFee::new(
-            fee::Milli::integral(43946000),
-            fee::Milli::integral(155381000000),
-        ),
+        fee_policy: fee::LinearFee::new(fee::Milli::new(43, 946), fee::Milli::integral(155381)),
```
Comment (NicolasDP) [inline exe-common/examples/generate_genesis_data.rs]: The parameter are inversed and not correct I believe

Comment (NicolasDP) [reply exe-common/examples/generate_genesis_data.rs]: https://github.com/input-output-hk/rust-cardano/blob/master/cardano/src/fee.rs#L162

Comment (NicolasDP) [reply exe-common/examples/generate_genesis_data.rs]: The constant is the first parameter : https://github.com/input-output-hk/rust-cardano/blob/master/cardano/src/fee.rs#L117-L122

**Pull request merged**